Formula/netcdf.rb

```diff
@@ -26,6 +26,7 @@ class Netcdf < Formula
   depends_on "hdf5"
 
   uses_from_macos "curl"
+  conflicts_with "netcdf-mpi", because: "netcdf-mpi is a variant of netcdf, one can only use one or the other"
```

Comment:
```suggestion
  uses_from_macos "curl"

  conflicts_with "netcdf-mpi", because: "netcdf-mpi is a variant of netcdf, one can only use one or the other"
```